pipe_segment/models/bigquery_message_source.py

```diff
@@ -55,7 +56,7 @@ def __init__(
     def filter_messages(self, start_date: datetime, end_date: datetime) -> str:
         template = self.filter_template
         if template is None:
-            template = "{{ filter_field }} " + \
+            template = "DATE({{ filter_field }}) " + \
```

Comment:
Isn't this the name of the field? 🤔

Reply:
No, `DATE()` is the BQ function to convert the `{{filter_field}}` that could be a `timestamp`, `date` or `datetime` field to a `date` data type so that when between is applied the positions for the whole day (at any time) are included.